packages/babel-helper-replace-supers/src/index.js

```diff
@@ -39,7 +40,7 @@ const visitor = {
     // We do have to traverse the key, since it's evaluated in the outer class
     // context.
     if (path.node.computed) {
-      path.get("key").traverse(visitor, state);
+      path.get("key").traverse(path.context.opts, state);
```

Comment:
I'm trying to create a base visitor that will be merged into several other complete visitors. This one defines the class environment traversal behavior (skip instance fields, class methods, but traverse computed keys).

But, it needs to be recursive. This is the best way I've found to reference the complete visitor from the sub-visitor.

/cc @loganfsmyth

Reply:
Curious. It's been in there since v5, but doesn't seem to be used anymore...

Reply:
Done.